modules/core/test/v2/integration/wallets.ts

```diff
@@ -27,7 +27,7 @@ describe('V2 Wallets:', function() {
   describe('Per-coin tests', function() {
     /** enable tbtg once btg is in test **/
     // TODO enable add txlm when it's supported by the platform and IMS in test
-    const coins = ['tbtc', 'tbch', 'txrp', 'teth', 'tltc', 'trmg'];
+    const coins = ['tbtc', 'tbch', 'teth', 'tltc', 'trmg'];
```

Comment:
is this because of the txrp node being down?

Reply:
yeah, I think this can be re-enabled now.